src/main/java/org/openshift/jenkins/plugins/openshiftlogin/OpenShiftOAuth2SecurityRealm.java

```diff
@@ -257,11 +258,13 @@ public void initialize(HttpRequest request) throws IOException {
                                 }
                             });
                     GenericUrl url = new GenericUrl(serverPrefix + "/oapi/v1/users/~");
-
+                    
+                    // do self-sar here to see if you have the "admin" or "view" verb in the "build.openshift.io" api group on the "jenkins" resource 
```

Comment:
v1.2.1 is the latest release we've got:
https://github.com/openshift/origin/releases/tag/v1.2.1
and that's what the image is using:
https://github.com/openshift/jenkins/blob/master/1/Dockerfile#L36